### DISCUSSION TITLE
Gloria Patri again

### DISCUSSION THREAD

**Discussion by ajmalton** [DivinumOfficium/divinum-officium#2598] [General]
The commons and ordo specify &Gloria and leave it to the code to work out if the Glória Patri is to be said.

There are (unfortunately) two sub Gloria, one for missa/ and one for horas/, and they disagree.  

The horas code omits GP during the Triduum. The missa code omits it during Passiontide.  I think the former is wrong, and that the rule should be the same for the whole liturgy.

That's what I find in my 1962 and 1950 books.  It's right, isn't it? The Gloria Patri is not said from Passion Sunday I through to the beginning of the Paschal Vigil, anywhere in the liturgy, in all books.

**Comment by gregordick** [comment]
In the office it's still said in most places during Passiontide, with the complete omission during the Triduum only.  In Passiontide the only places in the office that it's omitted are at the invitatory and in responsories (whether long or short).  See e.g. https://archive.org/details/p2breviariumroma00cathuoft/page/n211/mode/2up.

**Reply by ajmalton** [reply]
Oh yes, I see now, and the methods Gloria1 and Gloria2 to distinguish.

Once again the three possibilities for locating differences (namely: in the code; in the [Rule] or (condition); or in the database filename structure) tripped me up.
